src/Shift-ClassInstaller.package/ShiftClassInstaller.class/instance/copyObject.to..st

```diff
@@ -0,0 +1,15 @@
+copying
+copyObject: oldObject to: newClass
+	| newObject |
+	
+	newObject := newClass isVariable
+		ifTrue: [ newClass basicNew: oldObject size ]
+		ifFalse: [ newClass basicNew ].
+
+	newClass allSlots do: [ :newSlot | oldObject class slotNamed: newSlot name ifFound: [ :oldSlot | 
+			newSlot write: (oldSlot read: oldObject) to: newObject ] ].
+
+	newClass isVariable
```

Comment:
This will work only for Object subclasses. For ProtoObject or nil subclasses it will cause DNU